pkgs/applications/audio/vcv-rack/default.nix

```diff
@@ -7,10 +7,8 @@ let
   # Others are downloaded with `make deps`. Due to previous issues with the
   # `glfw` submodule (see above) and because we can not access the network when
   # building in a sandbox, we fetch the dependency source manually.
-  pfft-source = fetchFromBitbucket {
-    owner = "jpommier";
-    repo = "pffft";
-    rev = "74d7261be17cf659d5930d4830609406bd7553e3";
+  pfft-source = fetchzip {
+    url = "https://vcvrack.com/downloads/dep/pffft.zip";
     sha256 = "084csgqa6f1a270bhybjayrh3mpyi2jimc87qkdgsqcp8ycsx1l1";
```

Comment:
Did you double check if this hash changed?

Reply:
The unpacked hash has probably not changed (calling `:b vcv-rack` in nix repl used my cached `/nix/store` version)

Reply:
You can put in whatever URL you want and the hash is not invalidated. The only way I know to test this is to change the hash and test of the old one is still generated.

Reply:
I have changed the hash first (I'm aware of this behavior). Here's what I did:

- change the hash to `"0000...00"` (same length)
- `nix repl -I nixpkgs=. '<nixpkgs>'`
- invoke `:b vcv-rack`
- Get the hash from failure message, and update the source
- in the repl `:r`
- then `:b vcv-rack`
- this gave me back a cached build path

Reply:
```
$ nix-prefetch-url --unpack https://vcvrack.com/downloads/dep/pffft.zip
084csgqa6f1a270bhybjayrh3mpyi2jimc87qkdgsqcp8ycsx1l1
```

👍